ludwig/features/text_feature.py

```diff
@@ -150,16 +151,21 @@ def feature_data(column, metadata, preprocessing_parameters: PreprocessingConfig
         ):
             preprocessing_parameters["computed_fill_value"] = preprocessing_parameters["unknown_symbol"]
 
+        sequences = column
+        prompt_template = preprocessing_parameters["prompt_template"]
+        if prompt_template is not None:
+            sequences = backend.df_engine.map_objects(sequences, lambda x: prompt_template.format(input=x))
+
         return build_sequence_matrix(
-            sequences=column,
+            sequences=sequences,
             inverse_vocabulary=metadata[f"{prefix}str2idx"],
-            tokenizer_type=preprocessing_parameters[f"{prefix}tokenizer"],
```

Comment:
Would you also be able to remove the rest of the [backwards compatibility workaround](https://github.com/ludwig-ai/ludwig/pull/3298/files#diff-a6c1d0bcc6c3f898718429631bcfe0fc6edcb07a8cd6dab926f514ed282bca2eR133-R141) (introduced in https://github.com/ludwig-ai/ludwig/pull/1859)?

Reply:
I'm not too familiar with this code tbh, maybe you can take it in a follow-up?